backend/src/game/newcards/new_card.py

```diff
@@ -0,0 +1,9 @@
+from abc import ABC
+
+
+class NewCard(ABC):
+    def __init__(self, name, cost, effect, footnote):
```

Comment:
The value of footnote should be `footnote=None`, this is due to the fact that often there will not always be a footnote so it can be an optional field. If the argument is but it will make it easier to display if it's an empty string instead of a None, this is bad design as you are using information relevant to visualizations about how you create the underlying data structures which means you're solving a problem before it exists and only works if the visualization does not change.